src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -4,16 +4,13 @@
     private String name;
     private String color;
 
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
+    public Machine(String name, String color) {
         this.name = name;
+        this.color = color;
     }
 
-    public String getColor() {
-        return color;
+    public Machine() {
```

Comment:
Do you use this constructor?

Reply:
Code doesn't work correctly without this constructor(

Reply:
Why?